packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -109,40 +123,48 @@ class AccessLegBody extends Component<Props, State> {
             showLegIcon={showLegIcon}
           />
           <S.LegDetails>
-            <S.StepsHeaderAndMapLink>
-              <S.StepsHeader
-                aria-expanded={expanded}
-                onClick={this.onStepsHeaderClick}
-              >
-                <Duration seconds={leg.duration} />
-                {leg.steps && <S.CaretToggle expanded={expanded} />}
-                <S.InvisibleAdditionalDetails>
-                  <FormattedMessage
-                    defaultMessage={
-                      defaultMessages["otpUi.TransitLegBody.expandDetails"]
-                    }
-                    description="Screen reader text added to expand steps"
-                    id="otpUi.TransitLegBody.expandDetails"
+            {hideDrivingDirections ? (
```

Comment:
Forgot to mention that, could you add a story where the driving directions are hidden?

Reply:
Added in [16d5df4](https://github.com/opentripplanner/otp-ui/pull/658/commits/16d5df44cab0d972e67efa40eb36cae46829e831)!